.github/workflows/welcome-newcomers.yaml

```
@@ -0,0 +1,40 @@
+name: first-interaction
+
+on:
+  issues:
+    types: [opened]
+  pull_request:
+    branches: [main]
+    types: [opened]
+
+jobs:
+  check_for_first_interaction:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/first-interaction@v1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          issue-message: |
+            Welcome to the Pantsbuild Community. This looks like your first issue here.
+
+            If you haven't already, come say hi on Slack.
+            Use [this link](https://docs.google.com/forms/d/e/1FAIpQLSf9zgf-MXRnVDJbrVEST3urqneq7LCcy0zw8qU-GH4hPMn52A/viewform?usp=sf_link)
+            for an invitation.
+
+            Thanks again, and we look forward to your next Issue/PR :smile:!
+
+          pr-message: |
+            Welcome to the Pantsbuild Community. This looks like your first PR here.
+
+            We are so grateful for each an every contribution, big or small.
+            So thanks for spending your time and energy making this project better.
+
+            If you haven't gotten a reply in one business week, feel free to gently comment "bump" to ping the project admins.
```

```suggestion
            If you haven't gotten a reply in a week, feel free to gently comment "bump" to ping the project admins.
```

I feel like the this qualifier is a bit confusing (e.g. which jurisdiction?), while just a flat 7 days seems reasonable. Even over "slow" periods like the week near US Thanksgiving or Christmas/New Year, there's likely to be _some_ people around.